src/Provider/MicrosoftResourceOwner.php

```diff
@@ -68,17 +78,19 @@ public function getLastname()
      */
     public function getName()
     {
-        return $this->response['name'] ?: null;
+        return $this->response['displayName'] ?: null;
     }
 
     /**
+     * @deprecated To be removed in 3.0
```

Comment:
I deprecated this as I cannot find any corresponding attribute in here https://docs.microsoft.com/en-us/graph/api/resources/user?view=graph-rest-1.0#properties